tensorflow/compiler/xla/g3doc/tutorials/jit_compile.ipynb

```diff
@@ -82,7 +82,6 @@
       "outputs": [],
       "source": [
         "import tensorflow as tf\n",
```

Comment:
I think you need to remove the "," at the end of this line as well. Otherwise, the Json is not valid.

Reply:
Ehy